README.md

```diff
@@ -1,2 +1,15 @@
-# demo-app
-Demo App
+# Demo App
+Demo Android app with PR [Danger](https://danger.systems/ruby) and [SonarQube](https://sonarcloud.io) checks. 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mathroule/demo-app/blob/6cdf29ea85d754c7a758d6d634306f22f00195cd/README.md#L2"></span>io -&gt; Io, co, do, go, ho, ion, lo, mo, no, o, so, to, yo, I, Ia, Id, If, Ii, In, Ir, Is, It, Iv, Ix, bio, oi, BO, Ibo, IE, IL, IMO, IP, IPO, IQ, ISO, Ito, Jo, KO, Po, RIO(spelling)</td>
    </tr>
  </tbody>
</table>